sklearn/feature_extraction/text.py

```diff
@@ -655,6 +665,11 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
         in the range [0.7, 1.0) to automatically detect and filter stop
         words based on intra corpus document frequency of terms.
```

Comment:
Maybe here add a comment saying that language-specific stop words can be retrieve for instance from NLTK.